### DISCUSSION TITLE
Is James a jerk, or what? ;)

### DISCUSSION THREAD

**Discussion by flowers9** [endless-sky/endless-sky#9042] [Lore]
With the default start, one of your earlier interactions with James can involve asking him how much he's willing to pay for passage.  He replies that that's a rude question.  But as a long-time captain, he should know that practically no one takes a job without first finding out how much it pays.  In fact, all standard jobs give the pay scale up front, in the job description.  And while he'll tell you how much he'll pay for his first job for you, you can't even ask on the followup missions, and none of the mission descriptions mention payment.

Personally, I think he should praise you for checking on the pay for the first mission, and all the missions should list the payment in their descriptions, just like standard jobs (which they're supposed to represent).

**Comment by Zitchas** [comment]
I'm inclined to agree. Trusting him to pay you appropriately might be appropriate for a later mission; but early on when you've only known him for a short while it seems a bit presumptuous to assume that he's already got our confidence and complete trust to the point where it's inappropriate to ask him how much he is going to pay us.
